src/Polly.Core/Simmy/MonkeyStrategyOptions.TResult.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Polly.Simmy;
+
+#pragma warning disable CS8618 // Required members are not initialized in constructor since this is a DTO, default value is null
+
+/// <summary>
+/// The options associated with the <see cref="MonkeyStrategyOptions"/>.
+/// </summary>
+/// <typeparam name="TResult">The type of result the retry strategy handles.</typeparam>
+public abstract class MonkeyStrategyOptions<TResult> : ResilienceStrategyOptions
+{
+    /// <summary>
+    /// Gets or sets the injection rate for a given execution, which the value should be between [0, 1].
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>. Either <see cref="InjectionRateGenerator"/> or this property is required.
+    /// When this property is <see langword="null"/> the <see cref="InjectionRateGenerator"/> is used.
+    /// </remarks>
+    [Range(MonkeyStrategyConstants.MinInjectionThreshold, MonkeyStrategyConstants.MaxInjectionThreshold)]
+    public double? InjectionRate { get; set; }
```

Comment:
```suggestion
    public double InjectionRate { get; set; } = 0.001
```

Basically, when I add chaos strategy I expect some default behavior. We can set the injection rate to a very low value, but it should not be no-op.

Reply:
I think that makes sense as long as the default value for the `Enabled` property is false, we don't want to inject any chaos inadvertently